src/index.js

```diff
@@ -1 +1,93 @@
+/* eslint-disable linebreak-style */
 import './style.css';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/oaP1XqMFPoB60pz4ScpD/scores/';
+
+const submitBtn = document.getElementById('submitBtn');
+
+// const constructJson = (name, score) => {
+//   score = Number(score);
+//   const user = {
+//     user: name,
+//     score,
+//   };
+//   return user;
+// };
```

Comment:
- [ ] Let's keep our code cleaner and neater by removing unnecessary comments. Kindly review the whole file and fix the issue

Reply:
Done.